### DISCUSSION TITLE
Manual entry of coordinates

### DISCUSSION THREAD

**Discussion by Vbsasuke26** [orbat-mapper/orbat-mapper#221] [Ideas]
Hello Kjell. I was wondering if it might be feasible to incorporate an additional feature allowing us to manually input latitude and longitude coordinates for the location of a unit, in addition to the current option of selecting it from the map. Occasionally, we have coordinates with a higher level of precision, and selecting the exact coordinates on the map can be a bit challenging. Thank you for considering this suggestion.

**Comment by kjellmf** [comment]
Excellent suggestion! What is your preferred input format? MGRS, (lat, long), (long, lat) or other?

**Reply by Vbsasuke26** [reply]
Thank you. 
I think (long , lat) in degree decimals will be ideal. :)

**Comment by kjellmf** [comment]
I've added #222 to keep track of this

**Comment by kjellmf** [comment]
This functionality has now been implemented. If you double click a position in the unit state panel an input field will appear. You can even switch between different input formats. 
![image](https://github.com/user-attachments/assets/34861928-94c5-4e18-90b9-8e028f9d3cfd)
